orchestration/templates/cron-workflow.yaml

```diff
@@ -24,3 +27,26 @@ spec:
                 parameters:
                   - name: gcs-prefix
                     value: '{{ template "argo.timestamp" }}'
+      {{- if .Values.smokeTest.enable }}
+      - name: send-slack-notification
+        container:
+          image: curlimages/curl:7.70.0
+          env:
+            - name: WORKFLOW_ID
+              value: '{{ "{{workflow.name}}" }}'
+            - name: WORKFLOW_STATE
+              value: '{{ "{{workflow.status}}" }}'
+            - name: SLACK_URL
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.smokeTest.slackUrl.secretName }}
+                  key: {{ .Values.smokeTest.slackUrl.secretKey }}
+          command: [curl]
+          args:
+            - -XPOST
+            - -H
+            - 'Content-type: application/json'
+            - -d
+            - '{"text": "Workflow $(WORKFLOW_ID) entered state: $(WORKFLOW_STATE)"}'
+            - $(SLACK_URL)
```

Comment:
Slack's newest webhooks mechanisms tie the target channel to the URL, so the URL is all we need here.